examples/vercel-ai-chatbot/app/actions.ts

```diff
@@ -40,7 +36,56 @@ export async function getChat(id: string, userId: string) {
   return chat
 }
 
-export async function removeChat({ id, path }: { id: string; path: string }) {
+export async function addChat({ id, title }: { id: string; title: string }) {
+  const session = await auth()
+
+  if (!session) {
+    return {
+      error: 'Unauthorized'
+    }
+  }
+
+  await kv.hset(`chat:${id}`, {
+    title,
+    id,
+    createdAt: Date.now(),
+    userId: session.user.id,
+    messages: []
+  })
+  await kv.zadd(`user:chat:${session.user.id}`, {
+    score: Date.now(),
```

Comment:
[nit] naming: `score` and `Date.now()` don't seem to match 

If this is unavoidable, might be worth a comment for what score means?
Similar for `member`

Reply:
yea these are [redis](https://redis.io/commands/zadd/) specific concepts - I'll add a commend